jest.config.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
   moduleNameMapper: {
     "^highlight.js$": `<rootDir>/node_modules/highlight.js/lib/index.js`,
   },
-  collectCoverage: true,
+  collectCoverage: !!process.env.GENERATE_JEST_REPORT,
```

Comment:
Nit, but could we hoist this up to something like

```js
const useCoverage = !!process.env.GENERATE_JEST_REPORT

// use elsewhere
```

Reply:
👍 done (at least I hope I did what you meant ;) )

Reply:
exactly :)